$OpenBSD: patch-test_test_integration_pending_rb,v 1.2 2012/06/29 16:40:31 jeremy Exp $

Fix hang when running the test suite using rubinius.  skip would be
better but rubinius uses ruby 1.8 stdlib and doesn't support it.

--- test/test_integration_pending.rb.orig	Wed Dec 31 16:00:00 1969
+++ test/test_integration_pending.rb	Tue Jun 26 10:44:55 2012
@@ -21,6 +21,7 @@ class TC_Integration_Pending < SQLite3::TestCase
 
   def test_busy_handler_outwait
     skip("not working in 1.9") if RUBY_VERSION >= '1.9'
+    return if defined?(RUBY_ENGINE) && RUBY_ENGINE == 'rbx'
 
     busy = Mutex.new
     busy.lock
